dash/dash.py

```diff
@@ -424,6 +424,9 @@ def __init__(
 
         self.logger.setLevel(logging.INFO)
 
+    def _get_config(self):
+        return self.config
```

Comment:
is this needed?

Reply:
Nope - that was an early attempt at a solution.  I'll delete it

Reply:
done